BraintreeCore/src/main/java/com/braintreepayments/api/AnalyticsClient.kt

```diff
@@ -157,7 +158,8 @@ internal class AnalyticsClient @VisibleForTesting constructor(
         if (authorization == null) {
             return
         }
-        val metadata = deviceInspector.getDeviceMetadata(context, sessionId, integration)
+        // TODO: Pass paypal_context_id
+        val metadata = deviceInspector.getDeviceMetadata(context, sessionId, integration, "payPalContextID")
```

Comment:
Do we have something equivalent to DeviceInspector in BT iOS?

Reply:
The equivalent to `DeviceInspector` on iOS is `Metadata`